monailabel/monailabel_HelloWorld_radiology_3dslicer.ipynb

```diff
@@ -0,0 +1,253 @@
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) MONAI Consortium  \n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
+    "you may not use this file except in compliance with the License.  \n",
+    "You may obtain a copy of the License at  \n",
+    "    http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "Unless required by applicable law or agreed to in writing, software  \n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
+    "See the License for the specific language governing permissions and  \n",
+    "limitations under the License. \n",
+    " \n",
+    "# MONAI Label Hello, World! - Radiology App and Slicer Example\n",
+    "\n",
+    "***The Active Learning Process with MONAI Label***\n",
+    "\n",
+    "In this notebook, we provide a hello world example of MONAI Label usecase. Using Spleen segmentation in Radiology app as the demonstration, 3D Slicer as the client viewer, we show how MONAI Label workflow serves as interacitve AI-Assisted tool for labeling CT scans. \n",
+    "\n",
+    "![workflow](./figures/monailabel_radiology_3dslicer/teaser_img.png)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Prepare MONAI Label\n",
+    "\n",
+    "### Prerequisites\n",
+    "- **Install MONAI Label** weekly preview release: \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install monailabel-weekly"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Check Install and All Sample Apps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!monailabel apps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.2 Download radiology app\n",
+    "Choose \"radiology\" as the app"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!monailabel apps --download --name radiology --output apps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.3 Download sample data\n",
+    "Download MSD Spleen dataset as the sample dataset using monailabel API. The task is the volumetric (3D) segmentation of the spleen from CT image. The segmentation of spleen is formulated as the voxel-wise 2-class classification. Each voxel is predicted as either foreground (spleen) or background. The dataset is from the 2018 MICCAI challenge.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!monailabel datasets --download --name Task09_Spleen --output datasets"
```

Comment:
For the path to the datasets, I would propose to check if `MONAI_DATA_DIRECTORY` is existing. If it does, use it instead of asking the user to make another download.
The code snippet below is commonly used by many notebooks
```
directory = os.environ.get("MONAI_DATA_DIRECTORY")
root_dir = tempfile.mkdtemp() if directory is None else directory
print(root_dir)
```

Reply:
Thanks. monailabel dataset download API has a logic similar to this, if use monailabel datasets. We probably do not need this.